pkg/command/service/generate.go

```diff
@@ -187,14 +180,23 @@ func createKSVC(ns string, index int) (string, string) {
 		},
 	}
 	fmt.Printf("Creating ksvc %s in namespace %s\n", service.GetName(), service.GetNamespace())
-	_, err := ksvcClient.Services(ns).Create(context.TODO(), &service, metav1.CreateOptions{})
+	ksvcClient, err := p.NewServingClient()
+	if err != nil {
+		fmt.Printf("Failed to create serving client: %s\n", err)
+	}
+	_, err = ksvcClient.Services(ns).Create(context.TODO(), &service, metav1.CreateOptions{})
 	if err != nil {
 		fmt.Printf("Failed to create ksvc %s in namespace %s : %s\n", service.GetName(), service.GetNamespace(), err)
 	}
 	return service.GetNamespace(), service.GetName()
 }
-func checkServiceStatusReady(ns, name string) error {
+
+func checkServiceStatusReady(p *pkg.PerfParams, ns, name string) error {
 	start := time.Now()
+	ksvcClient, err := p.NewServingClient()
```

Comment:
same question as above

Reply:
And here, I use the `ksvcClient` global variable in L46 and so that `checkServiceStatusReady` will not initialize it again.